package.json

```diff
@@ -6,20 +6,20 @@
   "scripts": {
     "cache-clean": "rm -rf ~/.electron && rm -rf dist",
     "postinstall": "node install.js",
-    "pretest": "npm run cache-clean && node install.js",
-    "test": "standard && tape test/*.js"
+    "pretest": "npm run cache-clean && npm --silent run postinstall",
+    "test": "tape test/*.js && standard"
```

Comment:
While debug logging and commenting things out I kept getting standard errors before the tests ran so I switched to lint after running the tests.

Makes things a little more forgiving from editors like Atom where the `console.log` code snippet has a trailing semicolon which standard fails on and blocks running the tests.